src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -41,8 +41,7 @@
  */
 .dshDashboardGrid__item--hidden {
   position: absolute;
```

Comment:
Is it necessary to apply this `position: absolute;` style here? In inspecting these elements, it looks like they're already getting this positioning from another style.

Reply:
It doesn't look like it is necessary. I'll remove this line.